packages/core/src/components/included.scss

```
@@ -0,0 +1,15 @@
+@use 'sass:map';
+
+$_all-included: ();
+
+@function is-included($key) {
+  @if not map.get($_all-included, $key) {
+    $_current-included: (
+      $key: true,
+    );
+    $_all-included: map.merge($_all-included, $_current-included) !global;
+    @return false;
+  }
+
+  @return true;
+}
```

💅🏻  [nitpick]

I'd change a few lines so that it's arguably clearer.

```suggestion
$_included: ();

@function is-included($key) {
  @if not map.get($_included, $key) {
    $_included: map.set($_included, $key, true) !global;
    @return false;
  }

  @return true;
}
```